src/adapter/adapter.js

```diff
@@ -1,7 +1,13 @@
 // @flow
 import youbora from 'youboralib';
 import {core} from 'kaltura-player-js';
-const {Error, MediaType} = core;
+const {Error, MediaType, DrmScheme} = core;
+
+const DrmSchemeName = {
+  FAIRPLAY: 'FairPlay',
+  PLAYREADY: 'PlayReadyClassic',
+  WIDEVINE: 'WidevineClassic'
```

Comment:
remove classic from widevine and playready

Reply:
removed (note it is different in android https://github.com/kaltura/playkit-android/blob/dev/playkit/src/main/java/com/kaltura/playkit/PKDrmParams.java)